hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -179,6 +181,7 @@ protected HoodieBackedTableMetadataWriter(StorageConfiguration<?> storageConf,
         .setBasePath(dataWriteConfig.getBasePath())
         .setTimeGeneratorConfig(dataWriteConfig.getTimeGeneratorConfig()).build();
     this.enabledPartitionTypes = getEnabledPartitions(dataWriteConfig.getProps(), dataMetaClient);
+    this.initializedPartitionTypes = new ArrayList<>(enabledPartitionTypes.size());
```

Comment:
I didn't see any usage of this variable.

Reply:
fixed it